packages/react/src/components/Popover/index.tsx

```diff
@@ -370,7 +370,6 @@ export const Popover: PopoverComponent = React.forwardRef(
         autoAlign && displayName && ['ToggletipButton'].includes(displayName);
       const isAllowedTriggerComponent =
         autoAlign &&
-        displayName &&
         !['ToggletipContent', 'PopoverContent'].includes(displayName);
```

Comment:
If `displayName` is undefined won't this throw a reference error?

Reply:
I test it, it's not giving any error. Also, find this on internet: 
_Reference errors occur only when you try to use a variable that hasn't been declared. Since displayName is declared, there’s no error._
It does not give a reference error because displayName is defined as a constant with the value undefined.